examples/mamba/mamba.py

```diff
@@ -439,40 +445,44 @@ def forward(self, input_ids: torch.Tensor, input_mask: torch.Tensor):  # [batch_
 class MambaDecoderLayer(nn.Module):
     def __init__(
         self,
-        config: MambaModelConfig,
+        model_config: MambaModelConfig,
```

Comment:
Hmm so far we've been calling them config (same as transformers). Can we stick to that? 🙏